.github/workflows/test.yml

```diff
@@ -80,10 +82,18 @@ jobs:
           check-latest: true
       - name: Run Go tests
         run: go test -covermode atomic -coverprofile coverage.txt $(go list ./... | grep -v third_party/)
+      - name: Workaround buggy Codecov OIDC auth
```

Comment:
Is there an upstream github issue we can link to so we can remove the workaround when it's fixed?

Reply:
I didn't think linking it in code would spam them but I could be wrong. I was thinking referring to it in a code comment would help remind us to keep an eye on it, since once #104 is closed we won't think about it again.